cli/commands/root.go

```
@@ -177,7 +176,8 @@ func init() {
 	viper.SetDefault("API_URL", "https://api.wundergraph.com")
 
 	rootCmd.PersistentFlags().StringVarP(&rootFlags.CliLogLevel, "cli-log-level", "l", "info", "sets the CLI log level")
-	rootCmd.PersistentFlags().StringVarP(&DotEnvFile, "env", "e", ".env", "allows you to set environment variables from an env file")
+	rootCmd.PersistentFlags().BoolVar(&disableEnvFile, "no-env-file", false, "disables loading environment variables from a file")
```

Is it required to have `--no-env-file` and `--env-file`? Who not just `--env-file` and by default `wunderctl up` uses `.env` and all others empty string? WDYT?

---

Now that we've changed it to default to an empty string, I think we can remove `--no-env-file`. What bothers me a bit is defaulting to `.env` for up. I'd be more comfortable with leaving it empty in all cases, then if it has a value and it fails to load we can report an error. Having two different defaults makes error handling a bit messy.

---

`.env` is very common for development exists since the early days of wundergraph. I'd not remove this default for up. Loading .env in development is best-effort.